src/github_api_client.ts

```diff
@@ -20,13 +21,14 @@ export class GithubAPIClient {
     const results = await Promise.all([
       this.requestUserActivity(username),
       this.requestUserIssue(username),
+      this.requestUserCodeReview(username),
```

Comment:
I did something, and I hope at leats this, isn't wrong.

Reply:
Use `requestUserActivity` as an example. It's a private method that performs a GraphQL query. I'm seeing `requestUserCodeReview` referenced, but I'm not seeing it defined anywhere.

Reply:
> Like, I am not sure how to get the information from github

View the [GraphQL](https://docs.github.com/graphql) docs or the [REST API](https://docs.github.com/rest) docs. You can use the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer) to experiment with queries.